common/changes/@itwin/imodels-client-management/fix-documentation_2023-08-24-09-44.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
```

Comment:
We do not have changelog files, please remove.

Reply:
Done